controllers/awsmachine_controller.go

```diff
@@ -85,7 +80,7 @@ func (r *AWSMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reter
 		return reconcile.Result{}, nil
 	}
 
-	logger = logger.WithName(fmt.Sprintf("machine=%s", machine.Name))
+	logger = logger.WithValues("machine", machine.Name)
```

Comment:
Same here, what if we remove this as well?

Reply:
I think we lose context here if we don't provide the associated Machine.

Reply:
+1 for keeping